less/current/modules/searchbar.less

```diff
@@ -1,11 +1,17 @@
 .search-bar{
   box-sizing: border-box;
   float: right;
-  width: 290px;
+  width: 292px;
```

Comment:
increasing the width in 2px to fit the search button to which 2px of margin was added